app/screens/home/channel_list/categories_list/categories/categories.tsx

```diff
@@ -74,7 +81,7 @@ const Categories = ({
                     currentTeamId={teamId}
                     isTablet={isTablet}
                     onChannelSwitch={onChannelSwitch}
-                    onlyUnreads={onlyUnreads}
+                    onlyUnreads={showAllCategory}
```

Comment:
Only unreads only happen if `showOnlyUnreadsCategory`. Make sure you update the dependency of this callback.

Reply:
@larkox Updated